tsconfig.json

```
@@ -9,7 +9,6 @@
     "moduleResolution": "bundler",
     "lib": ["es2022", "dom", "DOM.Iterable"],
     "allowArbitraryExtensions": true,
```

Let's re-enable `skipLibCheck` to make the build work:

```suggestion
    "allowArbitraryExtensions": true,
    "skipLibCheck": true,
```